src/main/java/Ui.java

```diff
@@ -0,0 +1,77 @@
+// deals with interactions with the user
+// what the user type in
+// what will be shown to the user
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Ui {
+    Scanner scanner;
+
+    public Ui() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    void welcomeMessage() {
+        String logo = " ____        _\n"
+                + "|  _ \\ _   _| | _____\n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("What can I do for you today?");
+        System.out.println("Please tell me what to do!");
+        System.out.println("\n");
+        System.out.println("====================================================================================");
+    }
+
+    void terminateMessage() {
+        System.out.println("\n");
+        System.out.println("====================================================================================");
```

Comment:
Consider storing '==========================' as a constant as you would probably be using it mulitple times.